lib/TermUtil.ts

```diff
@@ -193,6 +193,51 @@ export function stringQuadToQuad<Q extends RDF.BaseQuad = RDF.Quad>(
   );
 }
 
+function escapeIRI(iriValue: string): string {
+  return iriValue.replace(escapeAll, replaceEscapedCharacter);
+}
+
+function escapeStringRDF(stringValue: string): string {
+  if (escape.test(stringValue)) {
+    stringValue = stringValue.replace(escapeAll, replaceEscapedCharacter);
+  }
+  return stringValue;
+}
+
+/* eslint-disable require-unicode-regexp */
+/* eslint-disable unicorn/better-regex */
```

Comment:
What change did this rule suggest?

Reply:
The linting automatically changes the first regex from
`/["\\\t\n\r\b\f\u0000-\u0019\uD800-\uDBFF]/` to
`/[\b\u0000-\u0019"\\\uD800-\uDBFF]/`.
They act the same. Is that because line feeds, carriage returns, and such are included in those unicode ranges? Nevertheless, the second pattern is even more cryptic than the first: I wouldn't know if a tab is included in that pattern or not, while that is clearer from the first.

Reply:
To be clear, the rearranged patterns do not affect the test outcomes. If the tests are complete and correct, this change doesn't matter, besides for clarity reasons.

Reply:
See the above comments. After rewriting the regex, the linter keeps quiet about it. :shrug: